### PR TITLE
Implement the `once` method on Evented.

### PR DESCRIPTION
**Pull request by hansthen**

### CODE REVIEW

File: folium/elements.py [hunk 1]
```diff
@@ -107,19 +107,20 @@ class EventHandler(MacroElement):
...
+            {{ this._parent.get_name()}}.{{ this.method }}(
                 {{ this.event|tojson}},
                 {{ this.handler.js_code }}
             );
         {% endmacro %}
         """
     )
 
-    def __init__(self, event: str, handler: JsCode):
+    def __init__(self, event: str, handler: JsCode, once=False):
```
Comment (Conengmo) [inline folium/elements.py:118]:
```suggestion
    def __init__(self, event: str, handler: JsCode, once: bool = False):
```

File: folium/map.py [hunk 2]
```diff
@@ -34,8 +34,14 @@ class Evented(MacroElement):
     """
 
     def on(self, **event_map: JsCode):
+        self._add(False, **event_map)
```
Comment (Conengmo) [inline folium/map.py:37]: I think it's nicer to use keyword arguments when the value doesn't inform what argument it's applying to.

```suggestion
        self._add(once=False, **event_map)
```

File: folium/map.py [hunk 3]
```diff
@@ -34,8 +34,14 @@ class Evented(MacroElement):
...
+
+    def once(self, **event_map: JsCode):
+        self._add(True, **event_map)
```
Comment (Conengmo) [inline folium/map.py:40]:
```suggestion
        self._add(once=True, **event_map)
```

File: folium/map.py [hunk 4]
```diff
@@ -34,8 +34,14 @@ class Evented(MacroElement):
...
+
+    def _add(self, once, **event_map: JsCode):
```
Comment (Conengmo) [inline folium/map.py:42]:
```suggestion
    def _add(self, once: bool, **event_map: JsCode):
```

**Review by Conengmo — APPROVED** [review]
Looks good! I have basically two comments, one about adding type hints, the other is more of a style thing. But nothing functional, so I'll approve it now already.

**Comment by hansthen** [comment]
> Looks good! I have basically two comments, one about adding type hints, the other is more of a style thing. But nothing functional, so I'll approve it now already.

Okay thanks for those.

**Pull request merged**